docs/application/native/guides/connectivity/nfc.md

```diff
@@ -98,9 +98,15 @@ The main features of the NFC API include:
 
   ![HCE routing](./media/nfc_hce_routing.png)
 
+  > **Note**
+  >
+  > - HCE operations begin with the launch of HCE application in Tizen Application Framework. With the execution of these operations, the data transfer to the application needs more time.
+  > - In case, you consider time efficiency, it is recommend not to create HCE applications and communicate with the Secure Elements provided in the device.
+  > - Alternatively, ensure that the HCE application is always in the launch state.
+
 ## Prerequisites
 
-To enable your application to use the NFC functionality:
+To enable use of NFC functionality in your application:
```

Comment:
@jh8801jung
1. Change
To enable the use of NFC functionality in your application:

with 
 
To enable your application to use NFC functionality:

Reason: To maintain consistency (after discussion) 



2. Global Comment
Remove the following line just below the diagram after **Note**

Pay attention to the following:

Reply:
@mijin85cho after mutual conclusion, the above changes are still mandatory ones. 

@jh8801jung kindly do the needful ASAP